storage/local/chunk/chunk.go

```diff
@@ -276,6 +276,7 @@ type Chunk interface {
 	UnmarshalFromBuf([]byte) error
 	Encoding() Encoding
 	Utilization() float64
+	Len() int
```

Comment:
I guess it deserves documentation that it returns the number of chunks (and not the length in byte or something), and that the implementation might be expensize.

(And while you are on it, the "add" above in the doc comment needs to be capitalized.)